cmd/crossplane/core/core.go

```diff
@@ -348,27 +348,20 @@ func (c *startCommand) Run(s *runtime.Scheme, log logging.Logger) error { //noli
 	return errors.Wrap(mgr.Start(ctrl.SetupSignalHandler()), "cannot start controller manager")
 }
 
-// SetupProbes sets up the health and ready probes.
+// SetupProbes sets up the health and readiness probes, as needed.
 func (c *startCommand) SetupProbes(mgr ctrl.Manager) error {
 	// Add default readiness probe
 	if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
 		return errors.Wrap(err, "cannot create ping ready check")
 	}
 
-	// Add default health probe
-	if err := mgr.AddHealthzCheck("ping", healthz.Ping); err != nil {
-		return errors.Wrap(err, "cannot create ping health check")
-	}
```

Comment:
Should we leave these, even if we don't use them for now? I could see them being handy in future. Or even just handy so that folks can configure their Helm chart to use them if they want.

Reply:
yes, make sense to me, reverted these changes